### PR TITLE
Fix install docs for localhost.

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -94,23 +74,86 @@ read -r -d '' TOPOLOGY <<'EOF'
         "node_egress_nic": "eth0",
         "node_mesh_ip": "10.0.1.1",
         "node_mesh_nic": "eth1",
-        "etcd_master": true
+        "etcd_master": true,
+        "primary_node": true,
+        "network_node": true,
+        "hypervisor": true,
```
Comment (jackadamson) [inline docs/installation.md:80]: Has this JSON been cut off?

**Review by jackadamson — CHANGES_REQUESTED** [review]
The installer is running now, will comment further once that completes

File: deploy/ansible/deploy.yml [hunk 2]
```diff
@@ -30,37 +30,11 @@
       include: includes/topology_add_node.yml
       loop: "{{topology}}"
 
-    # If we are installing to localhost, and are not using a packaged version
-    # we need to special case where configuration files come from.
-    - name: Set configuration file location
-      set_fact:
-        "template_path": "/srv/shakenfist/venv/share/shakenfist/templates"
```
Comment (jackadamson) [inline deploy/ansible/deploy.yml:37]: `template_path` appears to still be used, without it `TASK [Copy libvirt template]` fails with
```
ansible.vars.hostvars.HostVarsVars object' has no attribute 'template_path'
```

File: docs/installation.md [hunk 3]
```diff
@@ -3,34 +3,35 @@ title: Installation
...
+* Have a cloudadmin account setup with passwordless sudo, and a ssh key in its authorized_keys file. This is an ansible requirement, although the exact username is configurable in the METAL_SSH_USER variable.
 
 Note that we used to recommend deployers run the installer from git, but we've outgrown that approach. If you see that mentioned in the documentation, you are likely reading outdated guides.
 
+Finally, Shaken Fist is run as root, so we install it as root as well. Make sure you're installing as the right user as you follow these instructions!
+
 First install some dependencies:
 
 ```bash
 sudo apt-get update
```
[the remark this answers is not in the document]
Comment (mikalstill) [reply docs/installation.md:24]: Its tempting, but I worry a bit about people jumping into the doc half way and missing a disclaimer.

File: docs/installation.md (hunk 1 above)
Comment (mikalstill) [reply docs/installation.md:80]: No, that's me failing to operate a computer during a cut and paste.

File: deploy/ansible/deploy.yml (hunk 2 above)
Comment (mikalstill) [reply deploy/ansible/deploy.yml:37]: Nice catch, thanks.

**Pull request merged**